src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -281,6 +281,8 @@ export const CheckoutPayment = ({
     sendDatadogAction('checkout_start')
     try {
       setIsDataLoading(true)
+      const quoteIds = await fetchSelectedQuoteIds()
```

Comment:
So the quote ids here might be stale? Before this PR I mean.
Like, there are newer quote ids connected to this user session (quote cart?) ?

Reply:
Yes, they apparently good be -- but I haven't dug into exactly why.

We try to refresh the graphql query after editing information but somehow it doesn't propagate all the way.